silk-workspace/src/main/scala/org/silkframework/workspace/activity/workflow/LocalWorkflowExecutor.scala

```diff
@@ -293,9 +303,16 @@ case class LocalWorkflowExecutor(workflowTask: ProjectTask[Workflow],
                                        (implicit workflowRunContext: WorkflowRunContext): Unit = {
     val resolvedDataset = resolveDataset(datasetTask(workflowDataset), replaceSinks)
     try {
+      val registry = MeterRegistryProvider.meterRegistry
+      val stopwatch: Timer.Sample = Timer.start()
       executeAndClose("Writing", workflowDataset.nodeId, resolvedDataset, Seq(entityTable), ExecutorOutput.empty) { _ =>
         // ignore result
       }
+      stopwatch.stop(registry.timer(s"$prefix.workflow.dataset.execution", "dataset", workflowTask.id.toString))
```

Comment:
The operator metric uses the node Id, so I think it should be used here as well (workflowDataset.nodeId) instead of the task id.

Reply:
I added the **workflow dataset node ID** as a new tag pair.

Reply:
Now I get: `cmem_workflow_dataset_execution_seconds_max{app="di",dataset="Workflow_a01db34855e3bb9e",} 0.0642097`. Shouldn't this also include both the workflow and the node id?

Reply:
My bad. I changed the one below it. It should be fine now.